src/oca_github_bot/tasks/merge_bot.py

```diff
@@ -166,22 +167,19 @@ def _user_can_merge(gh, org, repo, username, addons_dir, target_branch):
 
 
 def _prepare_merge_bot_branch(
-    merge_bot_branch, target_branch, pr_branch, pr, username, merge_strategy
+    merge_bot_branch, target_branch, pr_branch, pr, username, merge_strategy, cwd="."
 ):
     if merge_strategy == MergeStrategy.merge:
         # nothing to do on the pr branch
         pass
     elif merge_strategy == MergeStrategy.rebase_autosquash:
         # rebase the pr branch onto the target branch
-        _git_call(["git", "checkout", pr_branch])
-        _git_call(
-            ["git", "rebase", "--autosquash", "-i", target_branch],
-            env=dict(os.environ, GIT_SEQUENCE_EDITOR="true"),
-        )
+        _git_call(["git", "checkout", pr_branch], cwd=cwd)
+        _git_call(["git", "rebase", "--autosquash", "-i", target_branch], cwd=cwd)
```

Comment:
This part of the code never runs, but... either you need to keep the `GIT_SEQUENCE_EDITOR` environment variable, or remove -i and --autosquash

Reply:
Hi, thanks for the review!
I'd like to keep the `--autosquash -i` options, but I have removed the `env` keyword because I don't think it is used since `_git_call` does not have such parameter.
Is it used in some way that I don't know?